mod_ci/controllers.py

```diff
@@ -101,6 +101,15 @@ def start_platforms(db, repository, delay=None, platform=None) -> None:
     vm_max_runtime = config.get("GCP_INSTANCE_MAX_RUNTIME", 120)
     zone = config.get('ZONE', '')
     project = config.get('PROJECT_NAME', '')
+    # Check if zone and project both are provided
+    if zone == "":
+        log.critical(f'GCP zone name is empty!')
+        return
+
+    if project == "":
+        log.critical(f'GCP project name is empty!')
+        return
```

Comment:
@Tarun-Arora Not in this PR, but do you think we should create an issue to create a mechanism to validate the configuration at the start of the platform? So, if any configuration is missing or inappropriate the user is notified when trying to start the platform, contrary to being informed when methods fail in runtime.